sample/WebHook-Generic/index.js

```diff
@@ -1,4 +1,7 @@
-﻿module.exports = function (body, context) {
+﻿module.exports = function (context, body) {
     context.log('Webhook was triggered!');
-    context.done(null, 'WebHook processed successfully! ' + body.a);
+    context.res = {
+        body: 'WebHook processed successfully! ' + body.a
```

Comment:
Note that WebHooks response is JSON by default, so we need sample/template to reflect that.

Also, I think this is one case where we do need special casing in the WebHook scenario to make it less SDK-like, even if it "breaks the pattern". e.g. I want to be able to simply write:

```js
module.exports = function (context, data) {
    context.output = { "doubleNum": data.num*2 };
    context.done();
}
```

or just

```js
module.exports = function (context, data) {
    context.done({ "doubleNum": data.num*2 });
}
```

This is the bread and butter json in / json out scenario that we really want to make as simple as possible.